test/libsolidity/Assembly.cpp

```diff
@@ -210,7 +210,7 @@ BOOST_AUTO_TEST_CASE(jump_type)
 			jumpTypes += item.getJumpTypeAsString() + "\n";
 
 	if (solidity::test::CommonOptions::get().optimize)
-		BOOST_CHECK_EQUAL(jumpTypes, "[in]\n[out]\n[in]\n[out]\n");
+		BOOST_CHECK_EQUAL(jumpTypes, "[in]\n[out]\n[out]\n[in]\n[out]\n");
```

Comment:
I haven't checked this, but I guess it's an ordinary jump to a "jump out" being inlined here.

Reply:
Here is the diff: https://gist.github.com/hrkrshnn/c0be59d40a918fc6a6a2d977b0dea5b3

An ordinary jump to `[swap2, swap1, pop, jump]` is inlined. We started inlining upto `4` bytes and this one would qualify. So I think the change is okay.